helix-term/src/commands.rs

```diff
@@ -227,7 +226,9 @@ impl Command {
         extend_line, "Select current line, if already selected, extend to next line",
         extend_to_line_bounds, "Extend selection to line bounds (line-wise selection)",
         delete_selection, "Delete selection",
+        delete_selection_noyank, "Delete selection (don't yank)",
         change_selection, "Change selection (delete and enter insert mode)",
+        change_selection_noyank, "Change selection (delete and enter insert mode, don't yank)",
```

Comment:
Let's instead use ", without yanking" for both of these.

https://github.com/mawww/kakoune/blob/master/src/normal.cc#L2236-L2239